tests/test_app.py

```diff
@@ -1,4 +1,228 @@
-import todo_app
+# from todo_app.data.trello_items import Item # <<< DONT KNOW WHY I CANT IMPORT THIS WITHOUT CAUSING TEST DISCOVERY ISSUES
+import pytest
+from todo_app.models.view_model import ViewModel
+from datetime import datetime, timedelta
 
-def test_thing():
-    assert True
\ No newline at end of file
+### ADDED BECAUSE CANT IMPORT TRELLO_ITEMS
+class Item:
+    def __init__(self, name, dateLastActivity = '', status = 'To Do', desc = '', ):
+        self.name = name
+        self.dateLastActivity = dateLastActivity
+        self.status = status
+        self.desc = desc
+
+@pytest.fixture
+def board_one_in_to_do():
+    test_item = Item(name='fred',status='To Do')
+    test_list=[]
+    test_list.append(test_item)
+    return test_list    
+
+@pytest.fixture
+def board_one_in_doing():
+    test_item = Item(name='fred',status='Doing')
+    test_list=[]
+    test_list.append(test_item)
+    return test_list    
+
+@pytest.fixture
+def board_one_in_done():
+    test_item = Item(name='fred',status='Done')
+    test_item.dateLastActivity = '123'
+    test_list=[]
+    test_list.append(test_item)
+    return test_list    
+
+@pytest.fixture
+def board_done_three_today_one_yesterday():
+    today=datetime.now()
+    yesterday=today-timedelta(days=1)
+    today_str = datetime.strftime(today,'%Y-%m-%dT%H:%M:%S.%fZ')
+    yesterday_str = datetime.strftime(yesterday,'%Y-%m-%dT%H:%M:%S.%fZ')
+    test_list=[]
+    i = 1
+    test_item = Item(name='Dick',status='Done',dateLastActivity=today_str)
+    while i <= 3:
+        test_list.append(test_item)
+        i += 1
```

Comment:
Typically this sort of looping/incrementing is condensed into a `for` loop rather than a `while` loop; technically they slightly differ because your existing code has i run from 1 to 3 and the new uses 0 to 2 but for us here that shouldn't matter (and if it does in future, we can adjust our for loop to account for it)
```suggestion
    test_item = Item(name='Dick',status='Done',dateLastActivity=today_str)
    for i in range(3):
        test_list.append(test_item)
```

Reply:
https://github.com/ianm1968/ToDoTake2/commit/74512fc9d1fc3af44d00616ee5245275b8b8a6c2